src/aurelia-authentication.js

```diff
@@ -35,8 +35,15 @@ function configure(aurelia, config) {
     });
   }
 
+  let client;
+
   // Let's see if there's a configured client.
-  let client = clientConfig.getEndpoint(baseConfig.current.endpoint);
+  if (baseConfig.current.endpoint) {
```

Comment:
actually, in the README i just said that ''=empty string would use default endpoint. thus i have to use endpoint !== null as '' should try to get the default and fail if there isn't. ergo, good catch. pays off sometimes to be persistent :smile: